packages/sst/src/constructs/NextjsSite.ts

```diff
@@ -88,17 +89,18 @@ export class NextjsSite extends SsrSite {
       handler = "index.handler";
     }
 
-    return new lambda.Function(this, `ServerFunction`, {
+    return new Function(this, `ServerFunction`, {
       description: "Server handler for Next.js",
       handler,
       currentVersionOptions: {
         removalPolicy: RemovalPolicy.DESTROY,
       },
-      logRetention: logs.RetentionDays.THREE_DAYS,
+      logRetention: 'three_days',
```

Comment:
why did remove the constant?

Reply:
`sst` has their own custom `Function` wrapper around `lambda.Function`. I updated `NextjsSite` to use that. The `API` is slightly different: https://github.com/serverless-stack/sst/blob/sst2/packages/sst/src/constructs/Function.ts